konfera/utils.py

```diff
@@ -70,6 +70,7 @@ def validate_email_template(raw_template, formatting_dict, required=True):
                                      'required:{} !~ formatting:{}'.format(required_keys, set(formatting_dict)))
         else:
             logger.warning('Not all required fields of the template were found in formatting dictionary.')
+            return raw_template
```

Comment:
Is there a possibility that we want unformatted/incomplete template to be shown in specific cases or can we remove this 'else' (and required parameter for this function? The other option is to convert the known parameters and unknown left untouched.